api/src/com/cloud/deploy/DeploymentClusterPlanner.java

```diff
@@ -55,7 +55,7 @@
             "true",
             "Enable/Disable cluster thresholds. If disabled, an instance can start in a cluster even though the threshold may be crossed.",
             false,
-            ConfigKey.Scope.Zone);
+            ConfigKey.Scope.Global);
```

Comment:
@koushik-das Ignore, the above comment. But, I think making this global takes away the option to override this at a zone level? Why remove this?